pkg/k8shandler/kibana/reconciler.go

```diff
@@ -363,6 +364,12 @@ func isDeploymentDifferent(current *apps.Deployment, desired *apps.Deployment) (
 		different = true
 	}
 
+	if current.Spec.Replicas != desired.Spec.Replicas {
```

Comment:
Please add a testcase for "isDeploymentDifferent"

Reply:
fixed